app/page-partials/my-dashboard/TeamInfoTabs/index.tsx

```diff
@@ -1,9 +1,8 @@
 import { useState, useEffect, useContext } from 'react';
 import { useRouter } from 'next/router';
 import styled from 'styled-components';
-import { Button as RequestButton, Tabs, Tab } from '@bcgov-sso/common-react-components';
-import Table from 'components/TableNew';
-import Button from 'html-components/Button';
+import { Tabs, Tab } from '@bcgov-sso/common-react-components';
```

Comment:
Will we eventually get rid of these components as well? Or are we stuck with them

Reply:
Yes, I think we should. But to keep this pr from totally blowing up in size I stopped at the button since the typography size was working with these